src/main/java/org/openrewrite/staticanalysis/EqualsToContentEquals.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2021 the original author or authors.
```

Comment:
Might want to adjust ./gradle/licenceHeader.txt and this here to set the right year.